x-pack/build_chromium/chromium_version.ts

```diff
@@ -0,0 +1,148 @@
+  // open node_modules/puppeteer/package.json
+  const puppeteerPackageJson: PackageJson = JSON.parse(
+    fs.readFileSync(path.resolve(REPO_ROOT, 'node_modules', 'puppeteer', 'package.json'), 'utf8')
+  );
+  const { version } = puppeteerPackageJson;
+  log.info(`Kibana is using Puppeteer ${version} (${forkCompatibilityMap[version]})`);
+  return forkCompatibilityMap[version];
+}
+
+/*
+ * Look at Puppeteer source code in Github for Kibana's version
+ * The src/revisions.ts file contains a revision string
+ */
+async function getChromiumRevision(kibanaPuppeteerVersion): Promise<ChromiumRevision> {
+  const url = `https://raw.githubusercontent.com/puppeteer/puppeteer/v${kibanaPuppeteerVersion}/src/revisions.ts`;
+  let body: string;
+  try {
+    log.info(`Fetching code from Puppeteer source: ${url}`);
+    const rawSource = await fetch(url);
+    body = await rawSource.text();
+  } catch (err) {
+    log.error(err);
+    throw new Error(`Could not fetch ${url}. Check the URL in a browser and try again.`);
+  }
+
+  let revision: ChromiumRevision | undefined;
+  const lines = body.split('\n');
+  let cursor = lines.length;
+  while (--cursor >= 0) {
+    // look for the line of code matching `  chromium: '0123456',`
+    const test = lines[cursor].match(/^\s+chromium: '(\S+)',$/);
+    if (test != null) {
+      [, revision] = test;
+      break;
+    }
+  }
+
+  if (revision == null) {
+    throw new Error(
+      `Could not find a Chromium revision listed in Puppeteer source! Check ${url} in a browser`
+    );
+  }
+
+  log.info(`Found Chromium revision ${revision} from Puppeteer ${kibanaPuppeteerVersion}`);
+  return revision;
+}
+
+async function getChromiumCommit(revision: ChromiumRevision): Promise<ChromiumCommit> {
+  const url = `https://crrev.com/${revision}`;
+  log.info(`Fetching ${url}`);
+  const pageText = await fetch(url);
+  const $ = cheerio.load(await pageText.text());
+  log.debug(`Page title: ${$('title').text()}`);
+
+  // get the commit from the page title
+  let commit: ChromiumCommit | null = null;
+  const matches = $('title')
+    .text()
+    .match(/\S{40}/);
+  if (matches != null) {
+    [commit] = matches;
+  }
+
+  if (commit == null) {
+    throw new Error(`Could not find a Chromium commit! Check ${url} in a browser.`);
+  }
+
+  log.info(`Found Chromium commit ${commit} from revision ${revision}.`);
+  return commit;
+}
+
+function showHelp() {
+  log.write(
+    dedent(chalk`
+      Display the Chromium git commit hash that correlates to a Puppeteer release.
+
+        - {dim Get Chromium commit for the current Puppeteer installation in Kibana:}
+          node x-pack/dev-tools/chromium_version
+
+        - {dim Get Chromium commit for Puppeteer v5.5.0:}
+          node x-pack/dev-tools/chromium_version 5.5.0
+
+      You can use https://omahaproxy.appspot.com/ to look up the Chromium release that first shipped with that commit.
+    `) + '\n'
+  );
+}
+
+async function chromiumVersion() {
```

Comment:
If you use `run()` from `@kbn/dev-utils` it will handle creating the tooling log for you, handle parsing logging level flags, and provides a standard interface for capturing errors and such. For more info checkout https://github.com/elastic/kibana/tree/master/packages/kbn-dev-utils/src/run